packages/back-end/src/models/ApiKeyModel.ts

```diff
@@ -351,7 +351,10 @@ export async function getAllApiKeysByOrganization(
   });
 
   return keys.filter((k) => {
-    return context.permissions.canReadSingleProjectResource(k.project);
+    return (
+      context.permissions.canReadSingleProjectResource(k.project) ||
+      k.userId == context.userId
```

Comment:
@akash-ind - To prevent any unintended bugs, we should use strict equality here (e.g. `k.userId === context.userId`).

Reply:
Got it

Reply:
@mknowlton89 Fixed this.